USAGE.md

```diff
@@ -1107,6 +1107,9 @@ memory.
 // move would place the cursor outside the plane.
 int ncplane_cursor_move_yx(struct ncplane* n, int y, int x);
 
+// Move cursor relatively to its current position.
+int ncplane_cursor_move_rel(struct ncplane* n, int y, int x);
```

Comment:
both of these probably ought be tagged with ((non-null, 1)). i'll do that though, no worries.